src/main/core/controller.c

```diff
@@ -472,13 +461,16 @@ gint controller_run(Controller* controller) {
         return 1;
     }
 
-    controller_updateMinTimeJumpNs(
-        controller, routinginfo_smallestLatencyNs(controller->routingInfo));
+    controller->minJumpTime =
+        routinginfo_smallestLatencyNs(controller->routingInfo) * SIMTIME_ONE_NANOSECOND;
+    info("using an initial minimum time jump of %" G_GUINT64_FORMAT " ns", controller->minJumpTime);
```

Comment:
> Don't we want to initialize `minJumpTime` to something else if `isJumpTimeDynamic` is true? Otherwise we would use the smaller of the dynamic jump time and the `routinginfo_smallestLatencyNs` jump time, which is not what we want.

When calculating `nextMinJumpTime`, it doesn't look at `minJumpTime` at all. It uses the minimum of `routinginfo_smallestLatencyNs()` and `nextMinJumpTime`, not `minJumpTime`, so the initial value of `minJumpTime` doesn't matter.

> I was going to suggest that we want to init to 0 if if `isJumpTimeDynamic` is true. But actually, I'm not sure what happens in that case for the first scheduling round. Maybe we instead want to initialize it to the max latency between any 2 pairs of nodes in the graph?

Setting to 0 will break the scheduler, it needs to be non-zero. Initializing it to the max latency would work, but would provide weird behaviour if the graph has a latency outlier. For example if one graph node has an edge with a latency of 3 seconds, I don't think we'd want to start the simulation with a scheduling interval of 3 seconds. Starting with the minimum latency makes sure that we're using reasonable scheduling intervals until the first packet is sent and `minJumpTime` is set.